requirements/base.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.12
```

Comment:
These should be generated with Python 3.8 still so we don't accidentally pull in any requirements that can't be run on Python 3.8

Reply:
@feanil  updated the files